### DISCUSSION TITLE
posts/ota-update/

### DISCUSSION THREAD

**Discussion by giscus** [basvandesande/azurecodingarchitect#9] [General]
# posts/ota-update/

Tinkering with Azure and other technologies

https://azurecodingarchitect.com/posts/ota-update/

**Comment by thombrink** [comment]
Hallo Bas, leuk artikel! Heb je de source code toevallig ergens op github staan?

**Reply by basvandesande** [reply]
Hi Thom,  
ik zal een passende versie op github zetten.  De code is wel een work in progress, maar je kan er wel de concepten uithalen.

**Comment by GunterO** [comment]
Hello,

Very interesting article, thank you for that. I'm wondering if you did the optimizations you mentioned in the article, or did some more stuff with it. I'm currently looking into the options for an ESP32 application which need OTA.
Because I'm more experienced in C# than C or Python, I'm considering the NanoFramework over ESP-IDF or MicroPython.
At the end of your article, you wouldn't chose the NanoFramework for production stuff. Has you opinion changed since then? Do you think it is matured in a way it is ready for production projects?

Thanks for your time and feedback!

Regards,
Gunter
